api/src/routes/filters.rs

```diff
@@ -283,7 +287,12 @@ pub struct FilterPost {
     )
 )]
 #[post("/filters")]
-pub async fn post_filter(db: web::Data<Database>, body: web::Json<FilterPost>) -> HttpResponse {
+pub async fn post_filter(
+    db: web::Data<Database>,
+    body: web::Json<FilterPost>,
+    user: Option<web::ReqData<User>>,
```

Comment:
Beautiful!

Reply:
yeah pretty happy with how that turned out, I'll rename stuff `current_user` or similar, just because `user` as is may conflict with other handlers where we have `user` variables